org.civicrm.bootstrapcivihr/scss/components/outside-namespace/_user-menu.scss

```diff
@@ -44,12 +39,19 @@
     margin: 0;
   }
 
+  li {
+    line-height: 50px;
+    white-space: nowrap;
+
+    > a,
+    > span {
+      padding: 0 30px 0 20px;
```

Comment:
I think this should be a var, you use it twice in this file. 

P.S. I would recommend to use the suffix smth like "-gap" and not "-padding" so you can use it later with, for example, margins.